src/Groups/matrices/matrix_manipulation.jl

```diff
@@ -238,6 +238,42 @@ function ishermitian_matrix(B::MatElem{T}) where T <: FinFieldElem
    return true
 end
 
+# return (true, h) if y = hx, (false, nothing) otherwise
+# FIXME: at the moment, works only for fields
+function _is_scalar_multiple_mat(x::MatElem{T}, y::MatElem{T}) where T <: RingElem
```

Comment:
So, how about restricting the code then?
```suggestion
function _is_scalar_multiple_mat(x::MatElem{T}, y::MatElem{T}) where T <: FieldElem
```